Sources/SwiftBuildSupport/PackagePIFBuilder.swift

```diff
@@ -498,7 +500,15 @@ public final class PackagePIFBuilder {
         settings[.WATCHOS_DEPLOYMENT_TARGET] = builder.deploymentTargets[.watchOS] ?? nil
         settings[.DRIVERKIT_DEPLOYMENT_TARGET] = builder.deploymentTargets[.driverKit] ?? nil
         settings[.XROS_DEPLOYMENT_TARGET] = builder.deploymentTargets[.visionOS] ?? nil
-        settings[.DYLIB_INSTALL_NAME_BASE] = "@rpath"
+
```

Comment:
This is a targeted fix, but we should try to mixup how we set sonames more generally on linux

Reply:
$ORIGIN is an appropriate value roughly equivalent to @rpath for ELF platforms